compiler/src/iree/compiler/Codegen/Utils/Utils.h

```diff
@@ -237,6 +237,14 @@ void sinkOpsInCFG(const SmallVector<Operation *> &allocs,
 // the inputs.
 bool hasFusedLeadingOp(linalg::LinalgOp rootOp);
 
+struct VscaleRange {
+  unsigned min;
+  unsigned max;
+};
```

Comment:
Would it make sense to make it has initial values (i.e., `0`) like what we have in `Passes.td`? In this case, perhaps we can avoid `std::optional`. Instead, we can pass `VscaleRange vscaleRange = VscaleRange{}` in the methods. What do you think?

Reply:
I'd prefer to keep the `std::optional`. I don't like encoding the semantics in the value, it's just the option parser does not support `std::optional<unsigned>`, so I have to do that in `Passes.td`.

Reply:
I see, makes sense to me. Thanks!